rust/connlib/tunnel/src/client.rs

```diff
@@ -281,37 +281,61 @@ fn send_dns_answer(
     }
 }
 
+/// A SANS-IO implementation of a client's functionality.
```

Comment:
```suggestion
/// A sans-IO implementation of a Client's functionality.
```
I think it's usually "sans-IO" or "Sans-IO" or "sans-I/O", 'sans' doesn't stand for anything https://sans-io.readthedocs.io/how-to-sans-io.html

Also in some parts of the docs we've been capitalizing Client, Gateway, Relay, etc., to show that they're short for the proper nouns "Firezone Client", "Firezone Relay", etc., and not just a network client or IPC client or anything else